File: uwp/ArtistWayUWP/Models/CheckinEntry.cs
using System.Collections.Generic;

namespace ArtistWayUWP.Models
{
    public sealed class CheckinEntry
    {
        // Chave = índice da pergunta (como string, "0", "1", "2"...).
        public Dictionary<string, string> Answers { get; set; } = new Dictionary<string, string>();
        public string SavedAt { get; set; } = "";
    }
}

File: android/ArtistWayAndroid/app/src/main/java/com/rodcarvalho/artistway/data/model/CheckinEntry.kt
package com.rodcarvalho.artistway.data.model

// Chave de `answers` = índice da pergunta (como string, "0", "1", "2"...).
data class CheckinEntry(
    val answers: Map<String, String> = emptyMap(),
    val savedAt: String = "",
)
